src/register.php

```diff
@@ -27,40 +27,27 @@
 				exit("This email is already registered");
 			}
 
-			// collecting information for validation
-			$time = time(); // in unix format
-			$validation = md5($email . $time . rand());
-			// ignore IP for now due to ip2long() not being able to handle IPv6 addresses
-			// TODO: remove this column from the database
-			$IP = 0;
-
-			$sth = $dice->dbconn->prepare("SELECT email FROM pending_validations WHERE email=?");
-			$sth->bind_param('s', $email);
-			$sth->execute() or trigger_error($mysqli->error);
+			// generating a random token for registration validation
+			// the DB table allows entries up to 32 chars
+			$validation = base64_encode(random_bytes(24));
```

Comment:
The token was made more secure (24 bytes instead of 16 bytes from md5) and no longer md5